range_sensor_layer/src/range_sensor_layer.cpp

```diff
@@ -21,8 +21,8 @@ void RangeSensorLayer::onInitialize()
   default_value_ = to_cost(0.5);
 
   matchSize();
-  min_x_ = min_y_ = -std::numeric_limits<double>::max();
-  max_x_ = max_y_ = std::numeric_limits<double>::max();
+  min_x_ = min_y_ = std::numeric_limits<double>::max();
+  max_x_ = max_y_ = std::numeric_limits<double>::min();
```

Comment:
Shouldn't this be std::numeric_limits<double>::lowest() ?
This relates to the reset happening in line 383. Imo, it should be lowest there as well.
min does not really make sense in this context.